assets/php/class-wgpb-block-library.php

```diff
@@ -396,10 +396,20 @@ public static function print_script_settings() {
 			'default_height'    => wc_get_theme_support( 'featured_block::default_height', 500 ),
 			'isLargeCatalog'    => $product_counts->publish > 200,
 		);
+
+		// Checkout settings.
+		$packages = WC()->shipping->get_packages();
+
+		$checkout_settings = array(
+			'isUserShopManager' => current_user_can( 'manage_woocommerce' ),
+			'isCouponsEnabled'  => wc_coupons_enabled(),
+			'isShippingEnabled' => ! empty( $packages ),
```

Comment:
I'm seeing an option for `WC()->shipping->enabled` here - can we use that or is packages better here?  Also not sure if all of `WC()->shipping` is being loaded in admin as other properties look empty:

<img width="734" alt="Screen Shot 2019-06-06 at 12 22 52 PM" src="https://user-images.githubusercontent.com/10561050/59026029-d34ac080-8855-11e9-88e3-6d04bb308402.png">

Reply:
I think this will work:
```php
$shipping_methods = WC()->shipping() ? WC()->shipping->load_shipping_methods() : array();
```